GVFS/GVFS.Service/Win/WindowsRequestHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System.Runtime.Serialization;
+
+namespace GVFS.Service.Handlers

```

Comment:
Two nits:

1. Use full "Windows" name for folder rather than "Win"
2. Update namespace to match folder structure -> `GVFS.Service.Windows`

(If this class should be inside the `GVFS.Service.Handlers` namespace then there should be a `Windows` folder inside GVFS.Service/Handlers that this file can be moved to).